src/Voron/Impl/Journal/WriteAheadJournal.cs

```diff
@@ -28,1625 +28,1626 @@
 
 namespace Voron.Impl.Journal
 {
-    public unsafe class WriteAheadJournal : IDisposable
-    {
-        private readonly StorageEnvironment _env;
-        private readonly AbstractPager _dataPager;
+  public unsafe class WriteAheadJournal : IDisposable
```

Comment:
It looks you changed the formatting - 2 spaces of indent instead of 4 what resulted in modifying _entire_ file. Please revert it.